src/components/Modal/Modal.stories.jsx

```diff
@@ -20,6 +20,41 @@ const Template = args => {
   );
 };
 
+const CrisisTemplate = args => {
```

Comment:
This template and the child content in `CrisisLineModal` are modeled after: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/6f11751e623e7d567c2f1b34f9176fdc224253d6/src/_components/html/crisis-line.html#L1